cmd/identities/get.go

```diff
@@ -29,10 +38,25 @@ func NewGetCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := cliclient.NewClient(cmd)
 
+			// we check includeCreds argument is valid
+			for _, opt := range includeCreds {
+				switch opt {
```

Comment:
Here you can use https://github.com/ory/x/blob/727727f69ff97272f0a0e2ccd6bd6e3d19c3bc6d/stringsx/switch_case_test.go#L59-L66 to have an automatically proper default error.

Reply:
Oh ok ty, as shown in https://github.com/ory/x/blob/727727f69ff97272f0a0e2ccd6bd6e3d19c3bc6d/stringsx/switch_case_test.go#L37, I moved out from a switch case into a single condition. As we only got one case